src/main/java/emu/grasscutter/game/systems/InventorySystem.java

```diff
@@ -833,6 +833,17 @@ public GameItem useItem(Player player, long targetGuid, long itemGuid, int count
                     break;
                 }
                 break;
+            case MATERIAL_BGM:
```

Comment:
Does unlocking from the music player actually do a `UseItemReq`? Last time I tried, they weren't usable from inventory.

Reply:
Yes. I saw `UseItemReq` in PacketSniffer of akebi

Reply:
![image](https://user-images.githubusercontent.com/60880668/194784798-9a91a746-d4c7-43e1-a2dd-b555fb5f8ebf.png)

After the confirm button was clicked, `UseItemReq` was appeared:
![image](https://user-images.githubusercontent.com/60880668/194784844-5edbf427-ad7e-4a2c-88e0-9b54abd05e10.png)